manual_qa_tests/workflows/dag_layout_test.py

```diff
@@ -0,0 +1,57 @@
+import aqueduct as aq
+
+NAME = "dag_layout_test4"
+DESCRIPTION = """
+    * Workflows Page: TODO: Describe layouts here.
+"""
+
+
+@aq.op
+def op1():
+    return 1
+
+
+@aq.op
+def op2(param):
+    return param
+
+
+@aq.op
+def op3(param1, param2):
+    return param1
+
+
+@aq.op
+def op4(param):
+    return param
+
+
+@aq.op
+def op5(param):
+    return param
+
+
+@aq.op
```

Comment:
I'd suggest change a few of them from `op` to `metrics` or `checks` given there are known issue and planned changes to improve the positioning of these nodes

Reply:
Sounds good, I've added some to the workflow.